### PR TITLE
settings: provide descriptions for color management settings

### PR DESCRIPTION
**Pull request by laurimyllari**
This is an update to the strings describing the color settings. There were a few marked with just TODO that are now explained.

Please feel free to adjust the wording or provide feedback if bigger changes are needed.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@jjd-uk for a check

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -18633,7 +18633,7 @@ msgstr ""
 #. Description of setting "System -> Video output -> 3D LUT" with label #36564
 #: system/settings/settings.xml
 msgctxt "#36565"
-msgid "TODO: 3D LUT filename description"
+msgid "Select a 3DLUT file to be used by default. If you place multiple 3DLUT files in the same directory, you can switch between them in the OSD menu during video playback. This allows for multiple display profiles to account for different viewing environments and source materials. A few ideas as starting points: gamma 2.2 for daytime viewing, 2.4 for nighttime; BT.601 primaries for SD material or older HD, BT.709 for modern HD; gamma input offset to avoid black crushing, output offset or absolute gamma for material with grey blacks."
 msgstr ""
```
Comment (jjd-uk) [inline addons/resource.language.en_gb/resources/strings.po]: I would change it to:

"Select **the** 3DLUT file to be used by default. If you place multiple 3DLUT files in the same directory**, then** you can switch between them in the OSD menu during video playback. This allows for multiple display profiles to account for different viewing environments and source materials. For example profiles for gamma 2.2 for daytime viewing, and 2.4 for night time."

A couple of nitpicks and no need for so many examples in my view, the wiki is the correct place for that.

File: addons/resource.language.en_gb/resources/strings.po [hunk 2]
```diff
@@ -18677,7 +18677,7 @@ msgstr ""
 #. Description of setting "System -> Video output -> Gamma mode" with label #36572
 #: system/settings/settings.xml
 msgctxt "#36573"
-msgid "TODO: Gamma mode description"
+msgid "Select the gamma curve formula. Use BT.1886 for standard with technical 2.4 gamma that accounts for display black and white levels and avoids perceptual clipping. On displays with higher than zero black the effective gamma will be lower. Use input offset for a similar curve with a manually specified effective gamma. Output offset allows for perceptual clipping, but can be used to compensate for incorrect mastering monitor settings that lead to too bright dark details. Absolute gamma does not account for display black at all and will clip the lowest levels to display black on any display with higher than zero black level."
 msgstr ""
```
Comment (jjd-uk) [inline addons/resource.language.en_gb/resources/strings.po]: "Use BT.1886 for standard with technical 2.4 gamma" doesn't read right for me, however I've no clue what message is trying to be conveyed so could just be my lack of technical understanding.

File: addons/resource.language.en_gb/resources/strings.po [hunk 3]
```diff
@@ -18699,7 +18699,7 @@ msgstr ""
 #. Description of setting "System -> Video output -> Lookup table size" with label #36576
 #: system/settings/settings.xml
 msgctxt "#36577"
-msgid "TODO: Lookup table size description"
+msgid "Specify the resolution for 3D lookup table. Use a lower resolution for quick preview and higher resolution for more accurate picture. Using a high resolution may take seconds to prepare when parameters are changed or a new video is started."
 msgstr ""
```
Comment (jjd-uk) [inline addons/resource.language.en_gb/resources/strings.po]: Minor nitpick "Specify the resolution for **the** 3D lookup table."

**Comment by jjd-uk** [comment]
Done my best to review considering my lack of knowledge on the technicalities, just some minor nitpicking if laurimyllari has the time to change, assuming he agrees with them.

**Comment by laurimyllari** [comment]
@jjd-uk thanks for the good feedback! I agree with them and will update accordingly.

**Comment by MartijnKaijser** [comment]
@jjd-uk could you update the PR so we can merge it?

**Comment by MartijnKaijser** [comment]
I meant @laurimyllari

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (laurimyllari) [inline addons/resource.language.en_gb/resources/strings.po]: I made the changes and shortened the end a bit.

File: addons/resource.language.en_gb/resources/strings.po (hunk 2 above)
Comment (laurimyllari) [inline addons/resource.language.en_gb/resources/strings.po]: I made it a bit more general. BT.1886 is a good default (as it is the current standard) - anybody choosing something different will hopefully know what they're doing. :)

File: addons/resource.language.en_gb/resources/strings.po (hunk 3 above)
Comment (laurimyllari) [inline addons/resource.language.en_gb/resources/strings.po]: Thanks, fixed!

**Comment by laurimyllari** [comment]
Updated

**Comment by MartijnKaijser** [comment]
@laurimyllari thx

**Pull request merged**